addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -376,6 +381,25 @@ def _get_operator(
                 )
                 return previous_operator_user
 
+        last_channel_by_operator = dict(
+            self.env["discuss.channel"]._read_group(
+                [
+                    ("livechat_operator_id", "in", users.partner_id.ids),
```

Comment:
Actually this is not working anymore. We should check member history instead. (chat bot forwarding to an operator for example).